### PR TITLE
Changes to dialogues and questlogs in new material in stoutford_tests

### PR DESCRIPTION
**Pull request by Rijackson**
More sprites added in items_rijackson_1.png. Clean up punctuation, typos etc. in dialogues, Change dialogue so that it is Andor's companion, not Andor himself, that persuades Glasforn to imprison the hero with the lich.

### CODE REVIEW

File: AndorsTrail/res/raw/conversationlist_halvor_surprise.json [hunk 1]
```diff
@@ -674,7 +675,7 @@
     },
     {
         "id":"halvor_bwmine_5",
-        "message":"Next on my list are some soft animal hairs. I think 4 should do. Would you find them for me ?",
+        "message":"Next on my list is some animal hair. It has ro be soft hair though. I think 4 handfuls should do. Would you find them for me?",
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_halvor_surprise.json]: -ro +to.
Can't believe I'm pointing out your typos :D

File: AndorsTrail/res/raw/conversationlist_halvor_surprise.json [hunk 2]
```diff
@@ -1086,11 +1087,11 @@
     },
     {
         "id":"halvor_mlake_3",
-        "message":"That's too bad. I will rest here until someone can heal me, if I can wait this long..."
+        "message":"That's too bad. I will rest here until someone can heal me If I can wait that long..."
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_halvor_surprise.json]: Punctuation ?

**Comment by Rijackson** [comment]
I just noticed a typo in my typo fixes. "There's now a path from the Duleian road to here that let's me avoid the maze." Should be "There's now a path from the Duleian road to here that lets me avoid the maze."

File: AndorsTrail/res/raw/conversationlist_stoutford.json [hunk 3]
```diff
@@ -1006,17 +1007,17 @@
...
                 "text":"N",
-                "nextPhraseID":"glasforn_rumblings60_6"
+                "nextPhraseID":"glasforn_rumblings60_5_1"
             }
         ]
     },
     {
         "id":"glasforn_rumblings60_6",
-        "message":"He gave me this, told me to wear it when I bring you to the cellar, and left. Take it. It scares me anyway. The monsters' attacks began shortly after.",
+        "message":"I\u2019m sorry! How was I to know you could actually kill it? You\u2019re just a kid! Anyway, now I can remove the necklace. Please take it. I never want to see it again.",
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_stoutford.json]: Unicode escapes ???

**Comment by Rijackson** [comment]
Also a typo I missed: "But don't tell this anybody else because I don't want to have some Feygard soldiers in front of my house one day." Should be "But don't tell this to anybody else because I don't want to have some Feygard soldiers in front of my house one day."

File: AndorsTrail/res/raw/questlist_stoutford.json [hunk 4]
```diff
@@ -6,31 +6,31 @@
...
+                "logText":"The priest's acolyte, Yolgen, told me that Andor was here a while ago, and since then, scary noises resonate in the church, seemingly coming from underground."
             },
             {
                 "progress":25,
                 "logText":"A member of the Thieve's Guild in the tavern of Stoutford told me that the tavern owner dealt with Andor."
             },
             {
                 "progress":30,
-                "logText":"Glasforn, the owner of the tavern of Stoutford, admits to knowing Andor, but evades all my questions by boasting about his beds' quality."
+                "logText":"Glasforn, the owner of the tavern of Stoutford, admits to knowing Andor, but evades all my questions by boasting about his bed's quality."
```
Comment (Zukero) [inline AndorsTrail/res/raw/questlist_stoutford.json]: It was meant to be a plural form...

Comment (Rijackson) [inline AndorsTrail/res/raw/questlist_stoutford.json]: I see I introduced a few new typos in the process! In my own defense, I needed several beers to get through more than two hundred lines of dialogue, so my brain was getting a little fuzzy by the end :D 
I have no idea how the escape characters got in there! I fix the errors, and give it one last review, and resubmit.

Comment (Rijackson) [inline AndorsTrail/res/raw/questlist_stoutford.json]: I fixed the typos you found, and the ones I found, and two more that neither of us found (until now).

Could you explain what you mean regarding the negated requirements. I sort of remember creating them, for reasons I'm sure seemed logical at the time. Is there a specific problem with them, or it's just that they are unusual?

File: AndorsTrail/res/raw/conversationlist_pathway_fallhaven.json [hunk 5]
```diff
@@ -150,7 +150,8 @@
                     {
                         "requireType":"questProgress",
                         "requireID":"pathway_fallhaven",
-                        "value":40
+                        "value":40,
+                        "negate":true
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_pathway_fallhaven.json]: Here.

File: AndorsTrail/res/raw/conversationlist_pathway_fallhaven.json [hunk 6]
```diff
@@ -187,7 +188,8 @@
                     {
                         "requireType":"questLatestProgress",
                         "requireID":"pathway_fallhaven",
-                        "value":30
+                        "value":30,
+                        "negate":true
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_pathway_fallhaven.json]: Here.

File: AndorsTrail/res/raw/conversationlist_shortcut_lodar.json [hunk 7]
```diff
@@ -12,7 +12,8 @@
                     {
                         "requireType":"questProgress",
                         "requireID":"lodar2",
-                        "value":50
+                        "value":50,
+                        "negate":true
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_shortcut_lodar.json]: Here.

File: AndorsTrail/res/raw/conversationlist_shortcut_lodar.json [hunk 8]
```diff
@@ -22,7 +23,8 @@
                     {
                         "requireType":"questProgress",
                         "requireID":"shortcut_lodar",
-                        "value":22
+                        "value":22,
+                        "negate":true
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_shortcut_lodar.json]: Here.

File: AndorsTrail/res/raw/conversationlist_shortcut_lodar.json [hunk 9]
```diff
@@ -56,7 +58,8 @@
                     {
                         "requireType":"questProgress",
                         "requireID":"shortcut_lodar",
-                        "value":40
+                        "value":40,
+                        "negate":true
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_shortcut_lodar.json]: Here.

File: AndorsTrail/res/raw/conversationlist_stoutford.json [hunk 10]
```diff
@@ -885,7 +885,8 @@
                     {
                         "requireType":"questProgress",
                         "requireID":"rumblings",
-                        "value":50
+                        "value":50,
+                        "negate":true
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_stoutford.json]: And here.

Comment (Rijackson) [inline AndorsTrail/res/raw/conversationlist_stoutford.json]: I'll look at them. I don't recall changing these, and can't think why I would have.

File: AndorsTrail/res/raw/conversationlist_glasforn_fixes.json [hunk 11]
```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "replies":[
+            {
+                "text":"N",
+                "nextPhraseID":"glasforn_rumblings60_5_3"
+            }
+        ],
+        "id":"glasforn_rumblings60_5_2",
+        "message":"I couldn\u2019t. When I tried, and failed, he laughed. He told me it was bound to that thing in the basement, which needed me to be its servant. "
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_glasforn_fixes.json]: More Unicode escapes

File: AndorsTrail/res/raw/conversationlist_glasforn_fixes.json [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    {
+        "replies":[
+            {
+                "text":"N",
+                "nextPhraseID":"glasforn_rumblings60_6"
+            }
+        ],
+        "id":"glasforn_rumblings60_5_4",
+        "message":"I didn\u2019t dare. The evening after Andor left his companion returned briefly. He warned me you might come here, and that if you found out what I had done you would kill me. He told me that my only hope was to make you the next victim."
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_glasforn_fixes.json]: There too.

File: AndorsTrail/res/raw/conversationlist_glasforn_fixes.json [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    {
+        "replies":[
+            {
+                "text":"I guess you are lucky I visited, although just asking me to kill it would have been easier. And nicer.",
+                "nextPhraseID":"glasforn_rumblings60_5_4"
+            }
+        ],
+        "id":"glasforn_rumblings60_5_3",
+        "message":"Then he laughed some more, and told me that all I had to do was kill that thing, and I would able to remove the necklace. I\u2019m no fighter though. I was too scared to even go near it."
```
Comment (Zukero) [inline AndorsTrail/res/raw/conversationlist_glasforn_fixes.json]: There too.

**Comment by Zukero** [comment]
Oh, and don't forget to add yourself in the way you see fit to the authors file ;)

File: AndorsTrail/res/raw/conversationlist_glasforn_fixes.json (hunk 13 above)
Comment (Rijackson) [inline AndorsTrail/res/raw/conversationlist_glasforn_fixes.json]: The unicode escapes and the negated requirements are very strange. I have no idea how they got there. I'll search for them, and make sure they are all gone.

**Pull request merged**